src/GeometryInstancing/Program.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System.IO;
+using System.Numerics;
+using Veldrid;
+using Veldrid.Sdl2;
+using Veldrid.StartupUtilities;
+using Veldrid.NeoDemo;
+
+namespace GeometryInstancing
+{
+    class Program

```

Comment:
Can this not extend `SampleBase` to avoid duplicating some of the boilerplate?

Reply:
Yes. I didnt do it because I'm writing my own engine so I wanted to do it from scratch. But I can for the sake of the tutorial